site/layouts/_default/_markup/render-codeblock-openapi.html

```
@@ -0,0 +1 @@
+{{ transform.Highlight (trim .Inner "\n") .Type }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)